microservice_a/Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+flask = "*"
+
+[dev-packages]
+requests = "*"
+
+[requires]
+python_version = "3.10"
+
+[scripts]
+server = "python main.py"
+test   = "python test.py"
```

Comment:
initially I thought just do docker cp, install dev libs and do test inside the same docker, but then I changed my mind